mcs/class/corlib/System.IO/Path.cs

```diff
@@ -971,6 +972,183 @@ public static ReadOnlySpan<char> GetFileName(ReadOnlySpan<char> path)
             return path;
         }
 
+		public static string Join(ReadOnlySpan<char> path1, ReadOnlySpan<char> path2)
```

Comment:
Add a marker where this was copied from

Reply:
This is already in a `#region Copied from corefx` (Added region to File.cs change too)